Beyond-legacy-code-guide.html

```
@@ -0,0 +1,1547 @@
+                    </li>
+                    <li>
+                        <a class="nav-a" href="#references"
+                            ><span class="n-num">09</span>参考情報源</a
+                        >
+                    </li>
+                </ul>
+            </nav>
+
+            <div class="main">
+                <div class="mobile-bar">
+                    <button id="navToggle" aria-label="目次を開く" aria-controls="sidebar" aria-expanded="false">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)